### PR TITLE
Update README.md

### LINKED ISSUE #4: Expand on README
Any chance you can make the README explain where `'some-secret'` should come from 

```rb
  # override application to decode token and allow only users with `tester` role
  def authorized_artsy_token?(token)
    decoded_token, _headers = JWT.decode(token, 'some-secret')
    decoded_token['roles'].include? 'tester'
  end
```

### PR DESCRIPTION
**Pull request by orta**
Fixes #4 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ class ApplicationController < ActionController::Base
 end
 ```
 
+# Decoding the JWT
+
+The JWT is signed using a different secret from the client secret for OAuth. For Artsy engineers: get it from the `internal_secret` on a `ClientApp` model. External engineers, send an email to ashkan@ or orta@ artsymail.com.
```
Comment (ashkan18) [inline README.md]: on `ClientApplication` model.

**Comment by ashkan18** [comment]
One # super minor change, thanks @orta for doing this!

**Comment by orta** [comment]
Done 👍

**Comment by ashkan18** [comment]
Awesome!

**Pull request merged**